projects/1.16/assets/xaeros-world-map/xaeroworldmap/lang/zh_cn.json

```
@@ -1,57 +1,120 @@
-  "gui.xaero_biome_colors": "原版模式生物群系",
-  "gui.xaero_wm_coordinates": "鼠标处坐标",
-  "gui.xaero_world_map_settings": "Xaero的世界地图设置",
-  "gui.xaero_world_map_screen": "世界地图屏幕",
-  "gui.xaero_texture_compression": "材质压缩",
-  "gui.xaero_flowers": "加载鲜花",
-  "gui.xaero_requires_reload": "需要重新加载世界才能生效！更低=更好的性能 ",
-  "gui.xaero_light_levels": "光照等级",
-  "gui.xaero_footsteps": "脚印",
-  "gui.xaero_terrain_slopes": "地形坡度",
-  "gui.xaero_terrain_depth": "地形深度",
-  "gui.xaero_update_chunks": "更新区块",
-  "gui.xaero_load_chunks": "加载新区块",
-  "gui.xaero_back": "返回",
-  "gui.xaero_vanilla": "普通",
-  "gui.xaero_accurate": "准确",
-  "gui.xaero_block_colours": "方块颜色",
-  "gui.xaero_lighting": "灯光",
-  "gui.xaero_off": "关",
-  "gui.xaero_on": "开",
-  "gui.xaero_debug": "调试",
-  "gui.xaero_open_settings": "打开设置",
-  "gui.xaero_open_map": "打开世界地图"
-}
\ No newline at end of file
+  "gui.xaero_quick_confirm": "快速手动确认",
+  "gui.xaero_default": "默认",
+  "gui.xaero_map_selection_box": "手动-每次切换世界或维度时，系统都会要求你选择并确认世界地图。 \n \n 世界出生-根据游戏客户端所知的世界重生点自动选择并确认世界地图。 如果服务器将原版指南针用于自定义功能，则可能会中断。 \n \n 单一-每个维度都有一张自动确认的世界地图。 对于只有一个世界的简单服务器很有用。\n\n 服务器-在服务端安装世界地图模组以使用此自动模式。",
+  "gui.xaero_select_map": "选择地图",
+  "gui.xaero_render_arrow": "渲染玩家箭头",
+  "gui.xaero_display_zoom": "显示缩放级别",
+  "gui.xaero_wm_ignore_heightmaps": "忽略服务器高度",
+  "gui.xaero_wm_box_ignore_heightmaps": "某些服务器在世界高度贴图中放置自定义物品，这可能会导致渲染不正确的贴图。有时，它甚至可以发生在高度更改的单人世界里。这个选项应该以牺牲性能为代价来解决这些问题。除非需要，否则不要使用。切换选项后重新进入世界！",
+  "gui.xaero_wm_waypoint_scale": "路径点范围",
+  "gui.xaero_wm_error_loading_properties": "加载服务器世界地图属性时出错。请重试。",
+  "gui.xaero_mw_server": "服务器",
+  "gui.xaero_mw_server_box": "地图选择由服务器自动控制。您可以在XaeroWorldMap目录中的特定于服务器的配置中使用“ignoreServerLevelId”关闭此功能。",
+  "gui.xaero_wm_next": "下一页 >>",
+  "gui.xaero_wm_previous": "<< 上一页",
+  "gui.xaero_wm_slopes_legacy": "旧版",
+  "gui.xaero_open_map_animation": "打开动画",
+  "gui.xaero_wm_slopes_default_3d": "默认 3D",
+  "gui.xaero_wm_slopes_default_2d": "默认 2D",
+  "gui.xaero_reload_everything": "重新加载区块",
+  "effect.xaeroworldmap.no_world_map": "没有世界地图",
+  "effect.xaeroworldmap.no_world_map_harmful": "没有世界地图",
+  "effect.xaeroworldmap.no_world_map_beneficial": "没有世界地图",
+  "gui.xaero_no_world_map_message": "该地图因药水效果而被禁用。",
+  "gui.xaero_box_zoom_in": "放大 \n (或使用鼠标滚轮)",
+  "gui.xaero_box_zoom_out": "缩小  \n (或使用鼠标滚轮)",
+  "gui.xaero_box_controls": "控制 \n \n 鼠标单击并拖动可滚动地图。 \n 滚动鼠标滚轮可放大和缩小。 \n 鼠标右键单击地图可以找到一些有用的快捷方式。 \n §2%2$s§r 打开全局设置。 \n §2%3$s§r 替代放大。 \n §2%4$s§r 替代缩小。 \n %1$s点击这里编辑按键绑定。",
+  "gui.xaero_box_controls_minimap": "\n 路径点控制： \n §2%s§r 创建一个新的路径点。 \n §2%s§r 快速创建一个临时路径点。 \n §2%s§r 切换路径点。 \n §2%s§r 切换渲染所有路径点。 \n §2%s§r 打开完整的路径点菜单。 \n 鼠标右键单击现有路径点以获得更多路径点控制。 \n \n ",
+  "gui.xaero_box_export": "将地图导出为一个 PNG 文件。",
+  "gui.xaero_export_confirm_1": "是否要将地图导出为 PNG 文件？",
+  "gui.xaero_export_confirm_2": "游戏在运行时会暂时冻结。",
+  "gui.xaero_box_map_switching": "地图切换选项",
+  "gui.xaero_box_open_waypoints": "路径点",
+  "gui.xaero_box_close_waypoints": "关闭路径点s",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "gui.xaero_box_close_waypoints": "关闭路径点",
```